src/bloxone_client/api_client.py

```diff
@@ -685,6 +685,9 @@ def path_param_value(
             If value_type is "resource_id", the value in the form of "app/type/id" or "type/id", the returned value will just have "id".
```

Comment:
please update the comment here, to include `integer_id`

Reply:
Done